File: frontend/model/Review.go
package model

type Review struct {
	MountainPathName string `json:"mountainPathName"`
	Vote             int    `json:"vote"`
	Title            string `json:"title"`
	Comment          string `json:"comment"`
	Author           string `json:"author"`
}

type ReviewForm struct {
	Vote    int    `form:"vote"`
	Title   string `form:"title"`
	Comment string `form:"comment"`
}

File: reviewManager/src/main/java/it/progetto/reviewmanager/model/Review.java
package it.progetto.reviewmanager.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
@Data
@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
public class Review {

    @Id
    @GeneratedValue
    public Long id;

    public String title;
    public String mountainPathName;
    public String comment;
    public Integer vote;
    public String author;

    @Override
    public String toString() {
        return "\"title\":\"" + title + "\",\"mountainPathName\":\"" + mountainPathName + "\",\"comment\":\"" + comment + "\"" + "\",\"vote\":\"" + vote + "\",\"author\":\"" + author + "\"";

    }
}
